go.mod

```diff
@@ -2,4 +2,16 @@ module gopkg.in/DataDog/dd-trace-go.v1
 
 go 1.12
 
-require github.com/tinylib/msgp v1.1.2
```

Comment:
Let's not forget to remove `go.mod` from the PR.

Reply:
I [reverted `go.mod` to `origin/v1`](https://github.com/DataDog/dd-trace-go/pull/721/commits/09f46e73c37ec3b45977a1e3be40abe8707ec37e) and CI [failed](https://app.circleci.com/pipelines/github/DataDog/dd-trace-go/508/workflows/9739a9c6-bb64-40f3-af39-eef4e5b0c052) again for the [same reason](https://github.com/DataDog/dd-trace-go/pull/721#issuecomment-688035434) (it tries to use a older version of the library that is missing some symbols).

I then [committed the `go.mod` obtained by just running `go test ./contrib/cloud.google.com/go/pubsub.v1`](https://github.com/DataDog/dd-trace-go/pull/721/commits/c7875c98c4b4905f08354d3d1c582dab00d6b10c), and CI [passed](https://app.circleci.com/pipelines/github/DataDog/dd-trace-go/509/workflows/4c8ae2d4-8e3c-4506-a1a8-cecea8b48dea).

Reply:
Not sure what you made of the previous discussion, but the bottom line is that we can't make any changes to `go.mod`. By "removing it from this PR" I just meant to undo any changes to it.

We'll have to figure out why CI is failing. I can't see it downloading `pubsub`, which is odd.

Reply:
I do see your point of specifying the minimum needed version in `go.mod`, and I think that do makes sense. If we were to do that, I would like to do it for _all_ integrations, not just some (starting with this one) and ideally treat it as a separate issue (I will strongly consider). But for now, I'd like to treat that issue separately and keep the old standard of not touching `go.mod`. We'll find a way to fix CI, even if it ends up being hacky, which will further motivate us to reconsider our approach. I hope this is ok with you.

Reply:
I guess the quick fix would be to simply `go get` it in the "Enforce some dependencies" section of CircleCI config.